app/(auth)/signup.js

```diff
@@ -66,6 +70,15 @@ export default function SignUp() {
         uniqueDrills: [],
       });
 
+      try {
+        await sendEmailVerification(auth.currentUser);
+        console.log("Verification Email Sent!");
+        showSnackBar("Verification Email Sent!");
+      } catch {
+        console.log("Error sending verification email: ", e);
+        showDialog("Error", getErrorString(e));
+      }
```

Comment:
It seems like the redirect event fires off before any of these logs or snackbars can be displayed
![image](https://github.com/user-attachments/assets/4d2ffac9-3fdd-4a77-a689-c3ef0b015ced)